app-config.yaml

```diff
@@ -253,33 +251,25 @@ catalog:
     # File locations are relative to the current working directory of the
     # backend, for example packages/backend/.
 
-    # Backstage example components
+    # Backstage example entities
     - type: file
-      target: ../catalog-model/examples/all-components.yaml
+      target: ../catalog-model/examples/all.yaml
     # Example component for github-actions and TechDocs
     - type: file
       target: ../../plugins/github-actions/examples/sample.yaml
     # Example component for TechDocs
     - type: file
       target: ../../plugins/techdocs-backend/examples/documented-component/catalog-info.yaml
-    # Backstage example APIs
-    - type: file
-      target: ../catalog-model/examples/all-apis.yaml
-    # Backstage example resources
-    - type: file
-      target: ../catalog-model/examples/all-resources.yaml
-    # Backstage example systems
-    - type: file
-      target: ../catalog-model/examples/all-systems.yaml
-    # Backstage example domains
-    - type: file
-      target: ../catalog-model/examples/all-domains.yaml
     # Backstage example templates
     - type: file
       target: ../../plugins/scaffolder-backend/sample-templates/all-templates.yaml
     # Backstage example groups and users
     - type: file
       target: ../catalog-model/examples/acme-corp.yaml
+      rules:
+      - allow:
+          - User
+          - Group
```

Comment:
maybe move this entire block up to the other one above so both catalog-model ones are next to each other